requirements.pip

```diff
@@ -0,0 +1,7 @@
+colorama==0.3.7
+future==0.15.2
+pluggy==0.3.1
+py==1.4.31
+pytest==2.9.0
+tox==2.3.1
+virtualenv==15.0.0
```

Comment:
how many of these are required for development, and how many are required simply to run the server?